src/app/shared/dynamic-form/types/form-element-base.ts

```diff
@@ -66,13 +70,15 @@ export class FormElementBase<T> {
     this.dependingOnValues = options.dependingOnValues;
     this.widget = options.widget;
     this.chips = options.chips ?? false;
+    this.allowClear = options.allowClear ?? false;
+    this.allowSelect = options.allowSelect ?? false;
     this.radioNewLine = options.radioNewLine ?? false;
     this.disabled = options.disabled ?? false;
   }
 }
 
 export class FormBase<T> {
-  id?: string;
+  id: string;
```

Comment:
will this not brake the code in another part? I think it was put id? for a reason but I can not remember :P